app/models/video.rb

```diff
@@ -1,9 +1,19 @@
 class Video < ActiveRecord::Base
   belongs_to :category
+  has_many :reviews, -> { order "created_at DESC" }
+  has_many :queue_items
 
   validates_presence_of :title, :description
 
   def self.search_by_title(search_term)
     search_term.blank? ? [] : where("title ILIKE ?", "%#{search_term}%").order(created_at: :desc)
   end
+
+  def average_rating
+    sum = 0
+    count = reviews.count.to_f
+    reviews.all.each { |review| sum += review.rating }
+    average = sum / count
+    average.round(1)
+  end
```

Comment:
There is a method called `.average` you could use to refactor this method. Check it out [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average).